src/index.html

```diff
@@ -7,12 +7,73 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="./styles/card.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/starsandrev.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/price.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/cardbuy.scss"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="Card Image"
```

Comment:
``alt`` attribute should describe the image content.

![image](https://github.com/mate-academy/layout_product-cards/assets/54860641/66dfbf87-3e69-437c-b15a-5d4cc56f27c0)